fastembed/embedding.py

```diff
@@ -74,11 +74,11 @@ def load_tokenizer(cls, model_dir: Path, max_length: int = 512) -> Tokenizer:
         return tokenizer
 
     def __init__(
-            self,
-            path: Path,
-            model_name: str,
-            max_length: int = 512,
-            max_threads: int = None,
+        self,
```

Comment:
Why does the diff have changes other than what we're changing? Please remove these formatting changes

Reply:
Those were from the pre-commit run.
https://github.com/qdrant/fastembed/blob/main/.pre-commit-config.yaml

Reply:
Aargh. The pre-commit config is outdated. Black needs to allow 120 line length. See pyproject.toml and Ruff config there.

Reply:
Can you please update the pre-commit config as well for a clean diff in this PR itself?

Reply:
Will do.

Reply:
Reverted the formatting.